controllers/topics.controllers.js

```diff
@@ -10,3 +10,14 @@ exports.getTopics = (req, res, next) => {
 exports.getEndpoints = (req, res) => {
   res.send({ endpoints: endpointsData });
 };
+
+exports.getArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  selectArticle(article_id)
+    .then((selectedArticle) => {
+      res.send({ article: selectedArticle });
```

Comment:
Even though it will default to a 200 I would be specific and add it in here personally